content/docs/user_guide.md

```diff
@@ -0,0 +1,765 @@
++++
+title = "User Guide"
+description = "In-depth instructions for using Kubeflow"
+weight = 10
+toc = true
+bref= "Detailed description of Kubeflow usage"
++++
+
+**Table of Contents**
+
+- [Using Kubeflow](#using-kubeflow)
+  - [Requirements](#requirements)
+  - [Deploy Kubeflow](#deploy-kubeflow)
+    - [Usage Reporting](#usage-reporting)
+    - [Bringing up a Jupyter Notebook](#bringing-up-a-jupyter-notebook)
+    - [Serve a model using TensorFlow Serving](#serve-a-model-using-tensorflow-serving)
+    - [Serve a model using Seldon](#serve-a-model-using-seldon)
+    - [Submitting a TensorFlow training job](#submitting-a-tensorflow-training-job)
+      - [Run the TfCnn example](#run-the-tfcnn-example)
+    - [Submitting a PyTorch training job](#submitting-a-pytorch-training-job)
+      - [Run the Pytorch MNIST example](#run-the-pytorch-mnist-example)
+  - [Advanced Customization](#advanced-customization)
+  - [Troubleshooting](#troubleshooting)
+    - [TensorFlow and AVX](#tensorflow-and-avx)
+    - [Minikube](#minikube)
+    - [RBAC clusters](#rbac-clusters)
+    - [OpenShift](#openshift)
+    - [Docker for Mac](#docker-for-mac)
+    - [403 API rate limit exceeded error](#403-api-rate-limit-exceeded-error)
+  - [Why Kubeflow Uses Ksonnet](#why-kubeflow-uses-ksonnet)
+
+
+# Using Kubeflow
+
+This guide will walk you through the basics of deploying and interacting with Kubeflow. Some understanding of Kubernetes, Tensorflow, and Ksonnet are useful in completing the contents of this guide.
+
+* [Kubernetes](https://kubernetes.io/docs/tutorials/kubernetes-basics/)
+* [Tensorflow](https://www.tensorflow.org/get_started/)
+* [Ksonnet](https://ksonnet.io/docs/tutorial)
+
+For an end to end example illustrating in details how to deploy kubeflow and run a training job from scratch, check out this [tutorial](https://ai.intel.com/lets-flow-within-kubeflow/).
+
+## Requirements
```

Comment:
Should we leave the sections on Requirements and Deploying Kubeflow out of this document. Instead, we could add link to the Getting Started section which should have up-to-date documentation on the latest methods for deployment.

Reply:
When I looked through the user guide, it actually has much more details for each command including debugging. Getting started section looks like the mini version of user guide. I don't know if removing this will be a good idea. What do you think?

Reply:
There is certainly some info here which is not captured in the other getting started docs. Speficially the portion about setting different local/cloud environments and being able to switch through them. Let's keep it for now then. I'll try to figure what's a better place to document that information.